src/components/carsoulSection/__test__/carsouelSection.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import CarsoulSection from './../carsoulSection';
```

Comment:
replace it with this
```suggestion
import CarsoulSection from '../carsoulSection';
```